server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/WebServiceUtils.java

```diff
@@ -219,6 +205,37 @@ public static void setConnectionConfiguration(DBPDriver driver, DBPConnectionCon
         }
     }
 
+    private static void setMainProperties(DBPConnectionConfiguration dsConfig, WebConnectionConfig config) {
+        if (config.getMainProperties() != null) {
+            for (Map.Entry<String, Object> e : config.getMainProperties().entrySet()) {
+                switch (e.getKey()) {
+                    case DBConstants.PROP_HOST -> dsConfig.setHostName(CommonUtils.toString(e.getValue()));
+                    case DBConstants.PROP_PORT -> dsConfig.setHostPort(CommonUtils.toString(e.getValue()));
+                    case DBConstants.PROP_DATABASE -> dsConfig.setDatabaseName(CommonUtils.toString(e.getValue()));
+                    case DBConstants.PROP_SERVER -> dsConfig.setServerName(CommonUtils.toString(e.getValue()));
+                    default -> throw new IllegalStateException("Unexpected value: " + e.getKey());
+                }
+            }
+            return;
+        }
+        if (!CommonUtils.isEmpty(config.getUrl())) {
```

Comment:
can we use here commonUtils.isNotEmpty?

Reply:
this line was before, but sure, we can use `isNotEmpty` here